packages/settings/src/settings.js

```diff
@@ -62,6 +62,17 @@ export const settings = {
      */
     FILTER_RESOLUTION: 1,
 
+    /**
+     * Override filter resolution by actually using a multisampling renderbuffer.
+     *
+     * @static
+     * @name FILTER_MULTISAMPLING
+     * @memberof PIXI.settings
+     * @type {boolean}
+     * @default true
+     */
+    FILTER_MULTISAMPLING: true,
```

Comment:
having this set to true by default, will this have a performance impact? If so, we should default to false.

Reply:
Oh, this was a vestigial part - since I’m not actually adding renderbuffer usage in the filter system. That’ll be a part of the filter system upgrade PR.